docker-for-azure/faqs.md

```diff
@@ -69,3 +69,34 @@ $ sudo ping 10.0.0.4
 ```
 
 Note that access to Docker for Azure and Azure happens through a shell container that itself runs on Docker.
+
+
+### What are the different Azure Regions?
+All regions can be found here: https://azure.microsoft.com/en-us/regions/
+An exerpt of the above regions to use when you create your service principal are:
+```
```

Comment:
The link shows the different regions, but doesn't actually provide the region code - If new regions are added, we can update it; this should be a good list for most.